dev/source/docs/ros2-pi.rst

```diff
@@ -95,3 +95,183 @@ The simplest solution is to wire an open telemetry port.
 
 .. figure:: ../images/PiZero2WTelem.png
    :target: ../_images/PiZero2WTelem.png
+
+..
+    Cosider documenting flow control setup, and also any configuration to enable /dev/ttyS0
+
+This allow GPIO13 and GPIO15 (/dev/ttySO) to communicate with ArduPilot.
```

Comment:
User also needs to run "sudo perl -pe 's/console=serial0,115200//' -i /boot/firmware/cmdline.txt" to disable the terminal on /dev/ttyS0

Reply:
I did not have to do that, mine was:
```
multipath=off dwc_otg.lpm_enable=0 console=tty1 root=LABEL=writable rootfstype=ext4 rootwait fixrtc cfg80211.ieee80211_regdom=GB
```

Do you think this was something different between ubuntu and raspberry PI OS?

Reply:
Ahh yes, that was a RasPiOS specific thing